src/utils/shell.ts

```diff
@@ -136,7 +136,7 @@ function execCore(cmd: string, opts: any, stdin?: string): Promise<ShellResult>
     return new Promise<ShellResult>((resolve, reject) => {
         const proc = shelljs.exec(cmd, opts, (code, stdout, stderr) => resolve({ code: code, stdout: stdout, stderr: stderr }));
         if (stdin) {
-            proc.stdin.end(stdin);
+            proc.stdin?.end(stdin);
```

Comment:
Do we need the if statement above now that we are using `?.` here?

Reply:
💡This was interesting if you see the `Line-138` it's `stdin` and **not** `proc.stdin` which is part of function param supplied, where as the line changed here is null check `proc.stdin`.

Reply:
Oops! I should have read that more carefully. You're right, different variables 👍